src/exporter/DaemonMetricCollector.cc

```diff
@@ -160,8 +160,11 @@ void DaemonMetricCollector::dump_asok_metrics() {
             labels.insert(multisite_labels_and_name.first.begin(), multisite_labels_and_name.first.end());
             counter_name = multisite_labels_and_name.second;
           }
-          auto perf_values = counters_values.at(counter_name_init);
-          dump_asok_metric(counter_group, perf_values, counter_name, labels);
+          auto counters_values_itr = counters_values.find(counter_name_init);
+          if (counters_values_itr != counters_values.end()) {
```

Comment:
Can you explain why this is needed?  The description of "exporter: check key exists before json::object at()" commit is empty...

Reply:
@idryomov  In an earlier iteration of this PR it was possible for the `counter dump` to not have all of the counters for a label that would have been in it's `schema dump`. Even though that mismatch is not possible in the current iteration, I thought it would be nice to keep this check since it is one of the only places where the `schema dump` output and `counter dump` output are crossed.

I recently retested this PR and reverted those changes after successful local testing.